data/campaigns/World_Conquest/scenarios/WC_II_scenario.cfg

```diff
@@ -60,7 +60,25 @@ _ "World Conquest 3p" #enddef
     [/event]
 #enddef
 
-#define WC2_CAMPAIGN_NEW PLAYERS
+# I know I can add them into the main campaign defines but it gets really messy if done that way so I prefer to keep my code clean
+
+#define ICON_ONE
+    "misc/blank-hex.png~BLIT(units/undead-necromancers/adept+female-defend-1.png~RC(magenta>brown)~CROP(10,0,72,72))~BLIT(units/dwarves/berserker/berserker-laugh-1.png~FL()~RC(magenta>white)~CROP(0,0,60,67),15,5)"
+#enddef
+
+#define ICON_TWO
+    "misc/blank-hex.png~BLIT(units/undead/spectre-se-attack-6.png~RC(magenta>darkred)~CROP(50,62,150,84),-15,-12)~BLIT(units/goblins/spearman-ne-defend.png~FL()~RC(magenta>darkblue)~CROP(0,0,60,67),20,15)"
+#enddef
+
+#define ICON_THREE
+    "misc/blank-hex.png~BLIT(units/drakes/armageddon-fire-se-4.png~RC(magenta>darkred)~CROP(10,0,72,72),-10,-10)~BLIT(units/goblins/spearman-ne-defend.png~FL()~RC(magenta>darkblue)~CROP(0,0,60,67),20,10)"
+#enddef
+
+#define IMAGE_ONE
+    "data/campaigns/The_Rise_Of_Wesnoth/images/story/trow_story_04-Fall_of_Eldaric.jpg~SCALE_INTO(800,600)"
```

Comment:
I think it's better to _copy_ this image into the World Conquest images directory and reference it from there instead. (Possibly this requires adding a `[binary_path]` declaration if it doesn't already have one?)

Also, there's only one of them, so I don't think it makes sense to call it `IMAGE_ONE`? I'd call it something like `WC_CAMPAIGN_IMAGE`; and while I'm on the subject, it might make sense to add a `WC_CAMPAIGN_` prefix to the icon names too.

Reply:
I only did like this since I was informed that WC might get customised image icons and campaign images at a later date. As of right now, it's a placeholder.